### DISCUSSION TITLE
ScriptReference/GUILayout.ExpandHeight

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#11] [Notes]
# ScriptReference/GUILayout.ExpandHeight

https://docs.unity3d.com/ScriptReference/GUILayout.ExpandHeight.html

**Comment by vertxxyz** [comment]
Content may expand to fill a space provided by [`MaxHeight`](https://docs.unity3d.com/ScriptReference/GUILayout.MaxHeight.html) if the provided `GUIStyle` has [`stretchHeight`](https://docs.unity3d.com/ScriptReference/GUIStyle-stretchHeight.html) set.
